.github/workflows/release.yml

```diff
@@ -1,71 +1,54 @@
-name: Release
-# This workflow is triggered when publishing a GitHub release
+name: sonar-release
+
 on:
   release:
     types:
       - published
 
-env:
-  PYTHONUNBUFFERED: 1
-
 jobs:
-  run_release:
+  release:
     runs-on: ubuntu-latest
-    name: Start release process
-    timeout-minutes: 60
+    name: Release
     steps:
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
           aws-access-key-id: ${{ secrets.BINARIES_AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.BINARIES_AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ secrets.BINARIES_AWS_REGION }}
-      - name: Run release action
-        id: run_release
+      - name: Release
+        id: release
         uses: SonarSource/gh-action_release/main@v4
         with:
           publish_to_binaries: true
-          slack_channel: sonarqube-build
+          slack_channel: team-sonarqube-build
         env:
           ARTIFACTORY_API_KEY: ${{ secrets.ARTIFACTORY_API_KEY }}
           BINARIES_AWS_DEPLOY: ${{ secrets.BINARIES_AWS_DEPLOY }}
           BURGRX_USER: ${{ secrets.BURGRX_USER }}
           BURGRX_PASSWORD: ${{ secrets.BURGRX_PASSWORD }}
-          CIRRUS_TOKEN: ${{ secrets.CIRRUS_TOKEN }}
-          PATH_PREFIX: ${{ secrets.BINARIES_PATH_PREFIX }}
           GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
-          RELEASE_SSH_USER: ${{ secrets.RELEASE_SSH_USER }}
-          RELEASE_SSH_KEY: ${{ secrets.RELEASE_SSH_KEY }}
           SLACK_API_TOKEN: ${{secrets.SLACK_API_TOKEN }}
-      - name: Log outputs
+      - name: Release action results
         if: always()
         run: |
-          echo "${{ steps.run_release.outputs.releasability }}"
-          echo "${{ steps.run_release.outputs.release }}"
-          echo "${{ steps.run_release.outputs.distribute_release }}"
-      - name: Notify success on Slack
-        uses: Ilshidur/action-slack@2.0.0
-        env:
-          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-        with:
-          args: "Release successful for {{ GITHUB_REPOSITORY }} by {{ GITHUB_ACTOR }}"
-      - name: Notify failures on Slack
-        uses: Ilshidur/action-slack@2.0.0
-        if: failure()
-        env:
-          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-        with:
-          args: "Release failed, see the logs at https://github.com/{{ GITHUB_REPOSITORY }}/actions by {{ GITHUB_ACTOR }}"
+          echo "${{ steps.release.outputs.releasability }}"
+          echo "${{ steps.release.outputs.promote }}"
+          echo "${{ steps.release.outputs.publish_to_binaries }}"
+          echo "${{ steps.release.outputs.release }}"
+
   maven-central-sync:
-    name: Sync with Maven Central
     runs-on: ubuntu-latest
+    name: Maven Central Sync
     needs:
-      - run_release
+      - release
     steps:
       - name: Setup JFrog CLI
-        uses: jfrog/setup-jfrog-cli@v2
+        uses: jfrog/setup-jfrog-cli@v1
```

Comment:
looks like the documentation is not up to date here, because on this part you had the correct one